tasks/Functions-client/request.js

```diff
@@ -101,7 +101,14 @@ task("functions-request", "Initiates a request from an Functions client contract
     const request = await buildRequest(requestConfig)
 
     // Estimate the cost of the request
-    const { lastBaseFeePerGas, maxPriorityFeePerGas } = await hre.ethers.provider.getFeeData()
+    let gasPriceEstimate
+    const { lastBaseFeePerGas, maxPriorityFeePerGas, gasPrice } = await hre.ethers.provider.getFeeData()
+    if (network.name === "mumbai") {
+      gasPriceEstimate = gasPrice.mul(lastBaseFeePerGas)
```

Comment:
wow, is that what we're supposed to do on Mumbai? Can you add a comment explaining why + a source link if any?

Reply:
I am also curious as well.  Why does Mumbai require this different estimation?